test/models/concerns/audit_extensions_test.rb

```diff
@@ -57,4 +57,45 @@ def setup
     end
     assert Audit.last.audited_changes.to_s.bytesize > 66000
   end
+
+  context "with multiple taxonomies" do
+    def setup
+      @loc = taxonomies(:location1)
+      @org = taxonomies(:organization1)
+    end
+
+    test 'records on create' do
+      domain = FactoryBot.create(:domain, :with_auditing, :locations => [@loc], :organizations => [@org])
+      audit = domain.audits.last
+      assert_equal 'create', audit.action
+      assert_equal [@loc.id], audit.location_ids
+      assert_equal [@org.id], audit.organization_ids
+    end
+
+    test 'records on update' do
+      domain = FactoryBot.create(:domain, :with_auditing, :locations => [], :organizations => [])
+      audit = domain.audits.last
+      assert_equal 'create', audit.action
+      assert_equal [], audit.location_ids
+      assert_equal [], audit.organization_ids
+
+      domain.name = 'blablabla' #needed for a new audit to be generated
+      domain.locations = [@loc]
+      domain.organizations = [@org]
+      domain.save!
+      audit = domain.audits.last
+      assert_equal 'update', audit.action
+      assert_equal [@loc.id], audit.location_ids
+      assert_equal [@org.id], audit.organization_ids
+    end
+
+    test 'records on destroy' do
```

Comment:
This test should fail until a new audited version is released containing collectiveidea/audited#413